### PR TITLE
Part of #18714: Replacing deprecated constants

### PR DESCRIPTION
**Pull request by dhruvv173**
## Explanation

- This PR is a part of #18714 
- #19526 
- #18896 
Made changes to the follow files

1. `ui/components/app/beta-header`
2. `ui/components/app/custom-spending-cap`
3. `ui/components/app/detected-token/detected-token-address`

### Before
image
image
### After
image
image
unit test
image

### Before
image
image
image

### After
image
image
image

unit test
image

### Before
image
image
### After
image
image

unit test
image

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Comment by dhruvv173** [comment]
hey @georgewrmarshall, I hope its fine if I can work on these issues till theres a fix for my build error. Could you please review this PR, thanks 🙏

File: ui/components/app/app-components.scss [hunk 1]
```diff
@@ -89,7 +89,6 @@
 @import 'advanced-gas-fee-popover/advanced-gas-fee-input-subtext/index';
 @import 'advanced-gas-fee-popover/advanced-gas-fee-defaults/index';
 @import 'currency-input/index';
-@import 'detected-token/detected-token-address/index';
```
Comment (georgewrmarshall) [inline ui/components/app/app-components.scss:92]: Removing unused CSS

**Comment by georgewrmarshall** [comment]
Hey @dhruvv173, Would it be possible to update this PR?

**Comment by dhruvv173** [comment]
> Hey @dhruvv173, Would it be possible to update this PR?

done!

**Review by georgewrmarshall — APPROVED** [review]
LGTM

**Comment by georgewrmarshall** [comment]
Hey @dhruvv173, looks like you have some failing e2e tests that need fixing

**Review by garrettbear — APPROVED** [review]

**Pull request merged**